### PR TITLE
Add module for Filer Shares

### PR DESCRIPTION
**Pull request by ygalblum**
Add UT

### CODE REVIEW

File: ansible_collections/ctera/ctera/plugins/modules/ctera_filer_share.py [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: ctera_filer_share
+short_description: CTERA-Networks Filer share configuration and management
```
Comment (saimonation) [inline ansible_collections/ctera/ctera/plugins/modules/ctera_filer_share.py:19]: Remove 'Networks'

File: ansible_collections/ctera/ctera/plugins/modules/ctera_filer_share.py [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+    - ctera.ctera.filer
+
+author:
+    - Saimon Michelson (@saimonation)
+    - Ygal Blum (@ygalblum)
+
+options:
+  state:
+    description:
+    - Whether the specified user should exist or not.
```
Comment (saimonation) [inline ansible_collections/ctera/ctera/plugins/modules/ctera_filer_share.py:34]: user?

File: ansible_collections/ctera/ctera/plugins/modules/ctera_filer_share.py (hunk 1 above)
Comment (ygalblum) [reply ansible_collections/ctera/ctera/plugins/modules/ctera_filer_share.py]: Done

File: ansible_collections/ctera/ctera/plugins/modules/ctera_filer_share.py (hunk 2 above)
Comment (ygalblum) [reply ansible_collections/ctera/ctera/plugins/modules/ctera_filer_share.py]: Good catch, done

**Pull request merged**